cmd/goQuery/commands/root.go

```diff
@@ -94,6 +94,19 @@ func init() {
 	rootCmd.Flags().DurationVarP(&cmdLineParams.QueryTimeout, "timeout", "", query.DefaultQueryTimeout, helpMap["QueryTimeout"])
 }
 
+var cliLogger *logging.L
+
+func initCLILogger() {
+	var err error
+	cliLogger, err = logging.New(logging.LevelError, logging.EncodingPlain,
```

Comment:
Plain logger is now taken for the error from the entrypoint. From my end, this is fine. I'll do the global-query one inside #43 .